README.md

```diff
@@ -143,6 +156,44 @@ jobs:
 
 Refer to the [Azure PowerShell](https://github.com/azure/powershell) GitHub Action to run your Azure PowerShell scripts.
 
+## Sample workflow that uses Azure login action with system-assigned managed identity
+
+```yaml
+# File: .github/workflows/workflow.yml
+on: [push]
+name: Azure System-assigned Managed Identity sample
+jobs:
+  build-and-deploy:
+    runs-on: self-hosted
+    steps:
+    - name: Azure Login
+      uses: azure/login@v1
```

Comment:
I am not sure if it is a good idea to implicitly use managed identity. As an alternative, we may add additional `with` arguments indicating `azure/login@v1` should use managed identity, such as `identity: true`. Better to confirm with our PMs.